pex/pep425.py

```diff
@@ -99,6 +99,14 @@ def _iter_supported_tags(cls, impl, version, platform):
     """
     # Predict soabi for reasonable interpreters.  This is technically wrong but essentially right.
     abis = []
+    if impl == 'cp' and version.startswith('2'):
```

Comment:
how about just checking for either '2' or '3' and reusing the abis.extend call + appending the 'abi3' bit to eliminate duplication?

something like:

```
if impl == 'cp' and (version.startswith('2') or version.startswith('3')):
  abis.extend(...)
  if version.startswith('3'):
    abis.append('abi3')
```